### ISSUE TITLE
[Enhancement] Allow instructors to unpin students

### ISSUE THREAD

**Issue by epfeiffe** [CarsonHoffman/office-hours-queue#8]
Currently this enhancement is not available. If an instructor pins a student on accident or needs to unpin them, they can't, and they are stuck at the top of the queue.

**Comment by CarsonHoffman** [comment]
I think this could be a good addition, but I would like to understand a bit better the workflow that leads to such a situation (just to get an overall understanding of how this might go wrong in other ways). The way I envisioned (and thus set up) the flow was for the "help" button to be used as the general tool to help students (it produces a pop-up on their screen that a staff member is coming), and that the "pin" button is used in exceptional circumstances (say: a staff member needing backup on an issue, or a student emailing half an hour later saying that nobody helped them, which we had a few hundred occurrences of in 280 🙃), after which the only reasonably resolution to a pin is to help the student. Does that follow your expectations and the issue is truly that you're accidentally pinning, or might a more targeted solution be to make the intended workflow clearer to prevent inadvertent pins in the first place?

**Comment by epfeiffe** [comment]
Upon further review, I believe I can use the stack and the frowny face feature to handle this and use the pin button in other circumstances as you mentioned. This issue can be resolved, thanks!

**Comment by y2kmurali** [comment]
To add some context, we occasionally run into platform-specific issues (e.g. vscode/xcode on mac, VS on windows, etc.) that some IAs that aren't as familiar with those specific platforms might not be able to answer. Our workflow on the other queue was to simply unpin the student which would return them to the top of the queue.

Our current workaround is to do what Emerson said above where we just have to manually pin them if we make the judgement call that they need to be helped again, and it's probably not too big a deal given that the workaround exists, but we do have to currently search through the "stack" to find that specific student to pin.

**Comment by CarsonHoffman** [comment]
Would a "pin" button on the after-help screen (with the "not helped") button help in this workflow? I can see how it would be annoying to look through the stack after spending a long time with the student on a busy queue.

**Comment by y2kmurali** [comment]
Yes, I think that solution would resolve this for sure. Might need to change the wording slightly instead of having buttons labeled "pin" and "not helped" to make sure that users understand what the difference is.
